docs/how-tos/provision-setup.md

```diff
@@ -0,0 +1,205 @@
+---
+title: "Complete setup for a machine"
+linkTitle: "Set a machine up"
+weight: 69
+type: "docs"
+description: "If you have received a machine that uses Viam and have been pointed to this guide, this guide will show you how to set it up."
+images: ["/platform/provisioning-demo.gif"]
+videos: ["/platform/provisioning-demo.webm", "/platform/provisioning-demo.mp4"]
+---
+
+If you have received a machine with Viam pre-installed on it, this guide will show you how to complete your device setup.
+
+{{< alert title="In this page" color="tip" >}}
+
+1. [Set up your machine using the Viam mobile app](#set-up-your-machine-using-the-viam-mobile-app).
+1. [Set up your machine using the captive portal](#set-up-your-machine-using-the-captive-portal).
+1. [Update machine configuration](#update-machine-configuration) to update hotspot details or add additional networks.
+   {{< /alert >}}
+
+## Prerequisites
+
+- Physical hardware constituting a machine
+- A laptop or mobile device if using the Viam mobile app
+
+## Set up your machine using the Viam mobile app
+
+{{<video webm_src="/platform/provisioning-demo.webm" mp4_src="/platform/provisioning-demo.mp4" alt="Using the Viam mobile app to provision a new machine with viam-agent." poster="/platform/provisioning-demo.jpg" max-width="400px" class="aligncenter imgzoom">}}
+
+{{< table >}}
+{{% tablestep %}}
+**1. Install the Viam mobile app**
+
+You can find the mobile app on the [App Store](https://apps.apple.com/vn/app/viam-robotics/id6451424162) and on [Google Play](https://play.google.com/store/apps/details?id=com.viam.viammobile&hl=en&gl=US).
+
+<a href="https://apps.apple.com/vn/app/viam-robotics/id6451424162" target="_blank">
+  <img src="https://github.com/viamrobotics/docs/assets/90707162/a470b65d-1b97-412f-9f97-daf902f2f053" width="200px" alt="apple store icon" class="center-if-small" >
+</a>
+
+<a href="https://play.google.com/store/apps/details?id=com.viam.viammobile&hl=en&gl=US" target="_blank">
+  <img src="https://github.com/viamrobotics/docs/assets/90707162/6ebd6960-08c5-41d4-81f9-42293fbfdfd4" width="200px" alt="google play store icon" class="center-if-small" >
+</a>
+
+{{% /tablestep %}}
+{{% tablestep %}}
+**2. Create a machine**
+
+Open the Viam mobile app and sign in.
+Then, select an organization and location for your machine.
+If you have already created a machine, select it.
+If you have not yet created a machine, click on **Add new smart machine** and give your machine a name.
+{{% /tablestep %}}
+{{% tablestep %}}
+**3. Connect to your machine's WiFi hotspot**
```

Comment:
I'm not sure this should be documented here. The mobile app will tell users when to switch networks, and they should follow the instructions there instead I think.

Reply:
I'll change the title. I think it's good to have for completeness. People may read through this guide without doing the actions at the time, just to get an overview.